content/tutorials/portenta-h7/vs-ard-lora/content.md

```diff
@@ -0,0 +1,157 @@
+
+* Use the [Arduino Web Editor](https://create.arduino.cc/editor) (this is the option this guide will follow)
+* Use the [Arduino IDE](https://www.arduino.cc/en/software), this way requires manual update of Arduino core and libraries.
+
+Plug the Vision Shield to the Portenta H7 and them to your PC through the usb port. Be sure to have selected the right board "Arduino Portenta H7 (M7 core)" and the right port.
+
+![vs-ard-select-port](assets/vs_ard_select_port.png)
+
+The LoRa module on the Vision Shield can be accessed by using the [MKRWAN library](https://github.com/arduino-libraries/MKRWAN). This library provides all the APIS to communicate with LoRa and LoRaWAN networks and can be Installed from the library Manager. The first code you need to upload and run is from the **MKRWAN** library, and its name is _FirstConfiguration_.
+
+![vs-ard-upload-code](assets/vs_ard_select_example.png)
+
+The only line you may need to change before uploading the code is the one that sets the frequency. Set the frequency code according to your country if needed. You can find more information about frequency by country at [this TTN link](https://www.thethingsnetwork.org/docs/lorawan/frequency-plans.html).
+
+```
+... // change this to your regional band (eg. US915, AS923, ...)
+ if (!modem.begin(EU868)) {    ...
+```
+
+[note]
+
+Please, consider that in Australia the boards connect correctly to TTN gateways on AS923 frequencies; AU915 frequencies requires the selection of sub band 2, not yet implemented in the firmware.
+
+[/note]
+
+Once the sketch completes upload, open the Serial Monitor. The following details will show:
+
+```
+Your module version is: ARD-078 1.1.9
+Your device EUI is: a8xxxxxxxxxxxx0a
+Are you connecting via OTAA (1) or ABP (2)?
+```
+
+### 4. Registring the Portenta on TTN
+
+Before your Portenta H7 can start communicating with the TTN you need to [register](https://www.thethingsnetwork.org/docs/devices/registration.html) it with an application. Go back to the TTN portal and scroll to the **Devices** section on your Application dashboard, then click on  **Register Device** button.
+
+![vs_ard_ttn_click_register](assets/vs_ard_ttn_click_register.png)
+
+On the registration page that shows up we must fill in two fields. The first one is the **Device ID** which this must be lowercase and without spaces. Since the ID should be unique, so you can use  the name of the board followed by the first and the last byte of the EUI. The second one is the **EUI** which is exactly the device EUI that appeared in the Serial Monitor. Copy paste the EUI from the Serial Monitor into the EUI field
+
+![vs_ard_ttn_register_device](assets/vs_ard_ttn_register_device.png)
+
+After pressing the Register button, your board will show up on the **Device Overview** page. You can now see all the information needed to complete the Arduino setup.
+
+![vs_ard_ttn_device_overview](assets/vs_ard_ttn_device_overview.png)
```

Comment:
@Lenardgeorge in the screenshot the app ID and the device ID don't match with the above description on how to name the device neither the previous screenshots